lab-jeff/test/test.js

```diff
@@ -0,0 +1,18 @@
+const expect = require('chai').expect;
+const net = require('net');
+
+const server0 = require('../lib/server0.js');
+
+server0.startServer();
+
+describe('chat server', function() {
+  it('should allow clients to message each other', function() {
+
+    let client1 = net.connect({port: 3000, host: 'localhost'});
+
+    client1.on('data', function(data) {
```

Comment:
This test is a false positive. It will pass even if I change the 'Welcome to the server Guest' text to something random.

It looks like you're setting up a listener on client1, but you never actually send any data to it in your test.